panacea/test/panacea/pml/parser_test.exs

```diff
@@ -22,6 +25,17 @@ defmodule Panacea.Pml.ParserTest do
         {:ok, {:process, [{:task, [{:action, [:tool, :script, :agent, :requires, :provides]}]}]}}
     end
 
+    test "it can parse all of jnoll's sample pml" do
+      {:ok, files} = File.ls(@fixtures_dir)
+      for file <-  files do
+        {:ok, pml} = Path.join(@fixtures_dir, file) |> File.read()
+
+        {status, possible_error} = Parser.parse(pml)
+
+        assert status == :ok, "failed to parse #{file}. error: #{inspect possible_error}"
```

Comment:
if the assertion fails, the string get's outputted. The normal error message from the generated lexer/parser is very unhelpful - typical erlang. If the parser/lexer reject the pml `possible_error` is `{line_number, lexer/parser module, error_string}`.